src/internal.c

```diff
@@ -3237,10 +3340,18 @@ int InitSSL(WOLFSSL* ssl, WOLFSSL_CTX* ctx)
         return MEMORY_E;
     }
 
+    /* FIPS RNG API does not accept a heap hint */
+#ifndef HAVE_FIPS
+    if ( (ret = wc_InitRng_ex(ssl->rng, ssl->heap)) != 0) {
```

Comment:
This is worth pointing out in the change for review. Having a new function addition to random.h that FIPS will not have.